### PR TITLE
Keyring vs USS keystore separation

### PR DESCRIPTION
**Pull request by stevenhorsman**
#### PR type
- Update user experience of using keyring vs USS keystores to make it clearer and easier

#### Relevant issues
Implements #1603

#### Changes proposed in this PR
- Add a new zowe-setup-keyring-certificates.env with the new parameters
- Tidy up the existing zowe-setup-certificates.env to remove the comments on the overloaded parameters
- Update zowe-setup-certificates.sh to be compatible with KEYSTORE_ALIAS being used for ZOWE_CERTIFICATE_LABEL
- Update tests

#### Is there a related doc issue or Pull Request? 
Doc issue number: https://github.com/zowe/docs-site/pull/1421

### CODE REVIEW

**Comment by stevenhorsman** [comment]
Keystore tests: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1654/8/
convenience tests: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1654/9/
ptf tests: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1654/11/

**Comment by stevenhorsman** [comment]
@vit-tomica @Joe-Winchester - I think I've captured what we discussed about separating the setup-certificate.env files to help the usability, so can you take a look at this PR and see what you think please?

**Comment by vit-tomica** [comment]
This is much better. Cert configuration will be much clearer now.

**Review by vit-tomica — APPROVED** [review]

File: bin/zowe-setup-keyring-certificates.env [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Location for generated certificates and/or JWT token
+KEYSTORE_DIRECTORY=/global/zowe/keystore
+# This variable has to be set to the Zowe certificate's LABEL specified in the JCL.
+ZOWE_CERTIFICATE_LABEL=localhost
+# Specify zowe user id to set up ownership of the generated certificates.
+# Set the variable to the same user id as in the jcl.
+ZOWE_USER_ID=ZWESVUSR
+# Specify zowe keyring that keeps zowe certificates, set
+# the variable to the same keyring that you used in the jcl.
+ZOWE_KEYRING=
```
Comment (Joe-Winchester) [inline bin/zowe-setup-keyring-certificates.env:30]: Would it be better if this defaulted to match the ZWEKRING JCL, so it becomes
`ZOWE_KEYRING=ZoweKeyring`

File: bin/zowe-setup-keyring-certificates.env [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+SSO_FALLBACK_TO_NATIVE_AUTH=true
+# optional - PKCS#11 token name for SSO. Must already exist
+PKCS11_TOKEN_NAME=
+# optional - PKCS#11 token label for SSO. Must not already exist
+PKCS11_TOKEN_LABEL=
+
+# Location for generated certificates and/or JWT token
+KEYSTORE_DIRECTORY=/global/zowe/keystore
+# This variable has to be set to the Zowe certificate's LABEL specified in the JCL.
+ZOWE_CERTIFICATE_LABEL=localhost
```
Comment (Joe-Winchester) [inline bin/zowe-setup-keyring-certificates.env:24]: In `ZWEKRING` JCL the parm is `SET   LOCALCA='localca'`
Should the `ZOWE_CERTIFICATE_LABEL=` match this so the default here should be
`ZOWE_CERTIFICATE_LABEL=localca` ?

**Review by Joe-Winchester — APPROVED** [review]

File: bin/zowe-setup-keyring-certificates.env (hunk 1 above)
Comment (stevenhorsman) [reply bin/zowe-setup-keyring-certificates.env]: Fixed

File: bin/zowe-setup-keyring-certificates.env (hunk 2 above)
Comment (stevenhorsman) [reply bin/zowe-setup-keyring-certificates.env:24]: I don't understand - the JCL has `SET    LABEL='localhost'`, so does match?

**Pull request merged**